### DISCUSSION TITLE
hyperlink for datatable

### DISCUSSION THREAD

**Discussion by adeguntoro** [yajra/laravel-datatables#3075] [Q&A]
i do follow this tutorial : https://yajrabox.com/docs/laravel-datatables/master/quick-starter

but how can i add hyperlink for spesirif
![jef](https://github.com/yajra/laravel-datatables/assets/26304962/9f89bd6b-f885-4deb-957c-36c1ef644587)
 column ( for example vendor number ), when i click Jefri name it will go edit page or update vendor page ?

found the answer :
![answer](https://github.com/yajra/laravel-datatables/assets/26304962/584fdc42-59c1-449a-99aa-4e6a7231d8fb)

**Comment by yajra** [comment]
You will also need to use rawColumns to display more columns with html values.
